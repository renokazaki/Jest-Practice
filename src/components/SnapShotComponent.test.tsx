import { render } from "@testing-library/react";
import SnapshotComponent from "./SnapshotComponent";

it("Snapshotテスト", () => {
  const { container } = render(<SnapshotComponent text="Hello, Snapshot2!" />);
  expect(container).toMatchSnapshot();
});
