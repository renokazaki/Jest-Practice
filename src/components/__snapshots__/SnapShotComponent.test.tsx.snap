// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Snapshotテスト 1`] = `
<div>
  <div>
    <p>
      Hello, Snapshot2!
    </p>
  </div>
</div>
`;
